static/src/stylesheets/layout/new-header/_menu-item.scss

```diff
@@ -25,6 +25,13 @@
         @include mq(desktop) {
             width: 100%;
         }
+
+        // hide * home links, which are duplicated on large screens
+        &:first-child {
```

Comment:
> These should always have a "home" link, otherwise it would be broken on small screens

Not sure if thats true in the case of News. News don't have a "news home" we just have headlines which isn't the same thing and there is always the potential for that to change.